pages/Settings.qml

```diff
@@ -437,6 +465,58 @@ Rectangle {
 
     }
 
+    // Choose blockchain folder
+    FileDialog {
+        id: blockchainFileDialog
+        title: "Please choose a folder"
+        selectFolder: true
+        folder: "file://" + persistentSettings.blockchainDataDir
+
+        onAccepted: {
+            var dataDir = walletManager.urlToLocalPath(blockchainFileDialog.fileUrl)
+            var validator = daemonManager.validateDataDir(dataDir);
+            if(!validator.valid) {
+
+                confirmationDialog.title = qsTr("Warning") + translationManager.emptyString;
```

Comment:
I try to avoid them in the qml files because that seemed to be the most common case when i started contributing. But since they are required in the c++ files it's easy to forget.